explore/beta/sasfit_compare.py

```diff
@@ -0,0 +1,666 @@
+from __future__ import division, print_function
+# Make sasmodels available on the path
+import sys, os
+BETA_DIR = os.path.dirname(os.path.realpath(__file__))
+SASMODELS_DIR = os.path.dirname(os.path.dirname(BETA_DIR))
+sys.path.insert(0, SASMODELS_DIR)
```

Comment:
Would vastly prefer that these sorts of `sys.path` manipulations were not used, even if it is only in tests/maintainer files. I can see significant hair loss associated with fiddling with sys.path like this in the future, when it accidentally picks up an old working copy, doesn't work inside a library.zip, needs to be called from setup.py as part of a release process or test script etc.

Reply:
Could address this by dropping an `__init__.py` into the explore and beta directories, so that these can be called as `python -m explore.beta.sasfit_compare`, etc.  from the sasmodels directory.  Doesn't seem worthwhile to me.